packages/modular-scripts/react-scripts/config/cdnStyleLoader.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+// This is a pitching loader, which means that it will exclude other loaders if its pitch method returns.
+// pitch methods are evaluated left to right and the correspondent default methods are evaluated later, right to left
+// For this reason, this loader must be first in the chain of other style loaders.
+// It will just set the output and interrupt if pitch returns and it will just act as a bypass of the chain output otherwise
+// If it existed another method of redirecting an external to a loader, it'd be simpler to just get rid of these pitch restrictions
+
+module.exports = function cdnStyleLoader(asset) {
+  // noop. Just return the translated asset if the pitching doesn't return (and the chain is processed)
+  return asset;
+};
+
+module.exports.pitch = function () {
```

Comment:
Being unfamiliar with pitching loaders and the details here, I do not claim to fully understand the implications of this implementation. Do you think the tests you have provided cover the edge cases that might occur?

Reply:
We decided to add a test for scoped packages in allow / block list. (test the behaviour of `filterDependencies` in `utils`)

Reply:
Done.